UPGRADE.md

```diff
@@ -56,7 +56,9 @@ The class with static methods has been converted to a service.
 The DCA configurations did not change, but the class changes are BC breaks:
 
 1. The `\Haste\Model\Relations` class has been converted to the `\Codefog\HasteBundle\DcaRelationsManager` service.
-2. The model class has been renamed from `\Haste\Model\Model` to `\Codefog\HasteBundle\Model\DcaRelationsModel`. 
+2. The model class has been renamed from `\Haste\Model\Model` to `\Codefog\HasteBundle\Model\DcaRelationsModel`.
+3. The following relation DCA keys has been deprecated as of 5.1: `referenceSql`, `fieldSql`, `tableSql`.
```

Comment:
```suggestion
3. The following relation DCA keys have been deprecated as of 5.1: `referenceSql`, `fieldSql`, `tableSql`.
```

Reply:
And why exactly? What's the difference, I cannot see it.

Reply:
Why deprecated? Because they contained the raw SQL data, which is incompatible now that we use Doctrine schema stuff.